pkg/landscaper/controllers/deployitem/actuator.go

```diff
@@ -0,0 +1,97 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package deployitem
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-logr/logr"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	lsv1alpha1helper "github.com/gardener/landscaper/apis/core/v1alpha1/helper"
+	"github.com/gardener/landscaper/pkg/landscaper/deployitem"
+	"github.com/gardener/landscaper/pkg/landscaper/operation"
+)
+
+func NewActuator(log logr.Logger, c client.Client, scheme *runtime.Scheme, pickupTimeout *int64) (reconcile.Reconciler, error) {
```

Comment:
can you add a doc string for the actuator and explain a bit what the controller is doing.